data/scripts/update_members.py

```diff
@@ -0,0 +1,96 @@
+import json
+with open('tmp/old-member-data','r') as f:
+    old_data = f.read()
+old_data_json = json.loads(old_data)
+with open('clubdata/requirement.tsv','r') as f:
+    new_data = list(filter(lambda x: "#N/A" not in x,[i.split('\t') for i in f.read().split('\n')[1:]]))
```

Comment:
Could you wrap the `i.split("\t")` in a namedtuple so future uses of the data are more legible (eg. instead of data[15], something like data.member_name)

It's fairly simple to use, just do

```python
from typing import NamedTuple

class Member(NamedTuple):
	field1: str
	field2: str
	... # rest of your fields

# say x is i.split("\t")
m = Member(i.split("\t"))
m[15] == m.member_name # can access with both index and prop name
# can also iterate over m like a tuple and dump to tsv like a tuple
```